src/config/firebase.js

```diff
@@ -3,7 +3,8 @@ import 'firebase/auth';
 import 'firebase/firestore';
 import 'firebase/storage';
 
-const config = {};
+const config = {
```

Comment:
Doesn't need to change
```suggestion
const config = {};
```